src/app/pages/ArticlePage/fixtureData.ts

```diff
@@ -50,6 +50,7 @@ const articleDataBuilder = (
   articleBlocksPopulator = blocksWithHeadlineAndText,
   atiAnalytics = {},
   type = 'article',
+  readTime = 0,
```

Comment:
Let's default it to a value > 0 - the common scenario will be that most articles will have a read time